.golangci.yml

```diff
@@ -134,6 +134,14 @@ issues:
     - text: "or a comment on this block"
       linters:
         - revive
+    # disabling to suppress deprecation warnings when updating OTLP to v0.8.0
+    # in support of breaking up the work into more manageable PRs
```

Comment:
We should create an issue, or document this in #3534 that is left to deal with for later.